Framework/include/QualityControl/Activity.h

```diff
@@ -22,6 +22,7 @@
 #include "QualityControl/ValidityInterval.h"
 
 #include "Rtypes.h"
+#include "BookkeepingApi/run.pb.h" // TODO beurk
```

Comment:
I am wondering if we should have a bunch of our structures have some interface to BK directly in their classes or better have one BK interface which can read/write all the concerned structures. If we do the latter, the advantages are:
- BK API will not be scattered everywhere in the QC code - one unit which communicates with BK. Easier to update the code related to BK if it is one place.
- probably faster compilation due to less inclusion of the BK headers